apps/laboratory/tests/shared/fixtures/w3m-verify-domain-mismatch-fixture.ts

```diff
@@ -0,0 +1,12 @@
+import type { ModalFixture } from './w3m-fixture'
+import { ModalPage } from '../pages/ModalPage'
+import { timingFixture } from './timing-fixture'
+
+export const testMVerifyDomainMismatch = timingFixture.extend<ModalFixture>({
```

Comment:
Why use a fixture here?

Reply:
Because we are making `ModalPage` inside the fixture and I need to supply custom config to it, requiring a different fixture.

I could make a new `ModalPage` using the same fixture, but then wouldn't it be doing unnecessary loading of the wrong page?